### PR TITLE
Add nested verify CPU models test case

### PR DESCRIPTION
**Pull request by qcheng-redhat**
ID: 1780520

Add nested verify CPU models test case

Signed-off-by: Qinghua Cheng <qcheng@redhat.com>

### CODE REVIEW

**Comment by qcheng-redhat** [comment]
@luckyh @yanan-fu @PaulYuuu @CongLi Please help review. Thanks!

File: qemu/deps/playbook.yml [hunk 1]
```diff
@@ -44,31 +44,28 @@
...
+         - name: Remove kar folder when bootstrap failed
+           file:
+             path: '{{kar_local}}'
+             state: absent
+         - name: Clean env when bootstrap failed 
+           pip:
+             name: aexpect, avocado-framework, avocado-framework-plugin-vt, avocado-framework-plugin-result-html
+             state: absent
 
     - fail:
```
Comment (PaulYuuu) [inline qemu/deps/playbook.yml:70]: I think need to move this task under `rescue`

File: qemu/tests/nested_test.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 from virttest import error_context
 from virttest import data_dir as virttest_data_dir
+from virttest import cpu as virttest_cpu
```
Comment (PaulYuuu) [inline qemu/tests/nested_test.py:11]:
```suggestion
from avocado.utils import cpu
```

File: qemu/deps/playbook.yml [hunk 3]
```diff
@@ -91,6 +88,19 @@
         msg: "Log dir is created"
       when: guest_log_dir.stdout != ""
 
+    - name: Get results.json file from guest vm
+      slurp:
+        src: '{{ guest_log_dir.stdout}}/latest/results.json'
+      register: results_json
+
+    - set_fact:
+        cancelled_case: "{{ item }}"
```
Comment (PaulYuuu) [inline qemu/deps/playbook.yml:97]:
```suggestion
        cancelled_case: "{{ (results_json.content|b64decode|from_json).cancel }}"
```

Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Fixed.

File: qemu/tests/nested_test.py (hunk 2 above)
Comment (qcheng-redhat) [reply qemu/tests/nested_test.py:11]: avocado.utils.cpu dosen't have get_host_cpu_models(). so keep using virttest.

File: qemu/deps/playbook.yml (hunk 1 above)
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Fixed.

File: qemu/deps/playbook.yml [hunk 4]
```diff
@@ -44,35 +44,31 @@
...
+           file:
+             path: '{{kar_local}}'
+             state: absent
+         - name: Clean env when bootstrap failed 
+           pip:
+             name: aexpect, avocado-framework, avocado-framework-plugin-vt, avocado-framework-plugin-result-html
+             state: absent
+         - fail:
+             msg: "bootstrap script failed"
+           when: bootstrap_result is failed
```
Comment (PaulYuuu) [inline qemu/deps/playbook.yml:71]: I believe it can be droped

File: qemu/deps/playbook.yml [hunk 5]
```diff
@@ -110,3 +117,9 @@
       shell:
         echo "Run test case failed"
       failed_when: run_case_result is failed
+
+    - name: Report test error
+      fail:
+        msg: "Some test cases are cancelled."
+      when:
+        cancelled_case > "0"
```
Comment (PaulYuuu) [inline qemu/deps/playbook.yml:125]: type(cancelled_case) = int, so `"0"` should not be a string
another question, do we need to handle skipped cases?

Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Fixed type issue. 

So far we don't have skipped cases, if need we could add handle skipped case.

File: qemu/deps/playbook.yml (hunk 4 above)
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Fixed. Thanks.

File: qemu/deps/playbook.yml [hunk 6]
```diff
@@ -110,3 +116,9 @@
       shell:
         echo "Run test case failed"
       failed_when: run_case_result is failed
+
+    - name: Report test error
+      fail:
+        msg: "Some test cases are cancelled."
+      when:
+        cancelled_case.cancel > 0
```
[the remark this answers is not in the document]
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: If do this change, non-passed test case will be filtered out. This filter is designed for cancelled test cases.  For error and failed test case will be filtered in the filter run_case_result.  We could adjust the value for expected cancelled cases, if needed.  What do other people think on this ?

File: qemu/deps/playbook.yml [hunk 7]
```diff
@@ -91,6 +86,17 @@
         msg: "Log dir is created"
       when: guest_log_dir.stdout != ""
 
+    - name: Get results.json file from guest vm
+      slurp:
+        src: '{{ guest_log_dir.stdout}}/latest/results.json'
+      register: results_json
+
+    - set_fact:
+        cancelled_case: "{{ results_json.content|b64decode|from_json }}"
```
[the remark this answers is not in the document]
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Fixed. Thanks.

File: qemu/deps/playbook.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Add a new param in_gating in .cfg, and handle cancelled test case as fail when in non-gating environment.

File: qemu/tests/cfg/nested_test.cfg [hunk 8]
```diff
@@ -16,3 +16,4 @@
         - boot_l2:
             case_name = boot
             install_node = yes
+        - check_cpu_model_l2:
```
Comment (PaulYuuu) [inline qemu/tests/cfg/nested_test.cfg:19]: Why a empty variants here. It's not a good idea.

File: qemu/deps/playbook.yml [hunk 9]
```diff
@@ -11,6 +11,7 @@
     bs_options: "{{bootstrap_options}}"
     host_log_dir: "{{host_log_files_dir}}"
     cmd_line: "{{command_line}}"
+    run_in_gating: "{{in_gating}}"
```
[the remark this answers is not in the document]
Comment (yanan-fu) [reply qemu/deps/playbook.yml:14]: @PaulYuuu  Yes, totally agree with you.
@qcheng-redhat  It is used to indicate if accept cancel, better use something like `accept_cancel`.  gating is only one usage scenario.

File: qemu/deps/playbook.yml [hunk 10]
```diff
@@ -110,3 +114,9 @@
       shell:
         echo "Run test case failed"
       failed_when: run_case_result is failed
+
+    - name: Handle cancelled test cases
+      fail:
+        msg: "Run tests in non-gating, handel cancelled test cases as failed"
```
Comment (yanan-fu) [inline qemu/deps/playbook.yml:120]: @qcheng-redhat  The msg need be updated too, It is handle cancelled test case according to different usage.

Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Updated fail msg, please review.

File: qemu/deps/playbook.yml (hunk 9 above)
[the remark this answers is not in the document]
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Change in_gating - > accept_cancel.

File: qemu/tests/cfg/nested_test.cfg (hunk 8 above)
Comment (qcheng-redhat) [reply qemu/tests/cfg/nested_test.cfg:19]: Case_name is passed by parameter to kar cmd, is there anything required here ?

File: qemu/deps/playbook.yml (hunk 10 above)
Comment (yanan-fu) [reply qemu/deps/playbook.yml]: @qcheng-redhat  `Run tests in non-gating environment, ***`.   This is not sutiable.
`non-gating` is only one usage.  others usages also can/can't accept cancel.
We should explain the problem itself, not one usage.

File: qemu/tests/cfg/nested_test.cfg (hunk 8 above)
Comment (yanan-fu) [reply qemu/tests/cfg/nested_test.cfg:19]: First, a empty variants is acceptable, means no special settings except the main part.
However, i do not think use if `case_name` exist to distinguish the nested test case is a good idea, what if there is new case for nested in the future ?
One possible method id to assign a name for the `variants`, @qcheng-redhat @PaulYuuu

Comment (PaulYuuu) [reply qemu/tests/cfg/nested_test.cfg:19]: @yanan-fu I know we can use an empty variants, but for this test case, it has specific steps like:
```python
            else:
                for i in host_cpu_models:
                    case_name = "x86_cpu_model." + i
                    cpu_model_case_names = cpu_model_case_names + case_name + ","
```
So it's not clear in cfg what this variant wants to do.

Comment (yanan-fu) [reply qemu/tests/cfg/nested_test.cfg:19]: @PaulYuuu  Yes, it use if `case_name` exist to distinguish the two test case, it not a good idea as i said before. Suggest to use variants name.

Comment (yanan-fu) [reply qemu/tests/cfg/nested_test.cfg:19]: May be define the `prefix_case_name = x86_cpu_model`. Then in the script, if the variants name is `check_cpu_model_l2`, generate the final case_name
```
for i in host_cpu_models:
    case_name += '%s.%s,' %(prefix_case_name, i)
```

File: qemu/deps/playbook.yml (hunk 10 above)
Comment (qcheng-redhat) [reply qemu/deps/playbook.yml]: Updated fail msg. Please review.

File: qemu/tests/cfg/nested_test.cfg (hunk 8 above)
Comment (qcheng-redhat) [reply qemu/tests/cfg/nested_test.cfg:19]: Fixed.

File: qemu/tests/nested_test.py [hunk 11]
```diff
@@ -62,13 +63,26 @@ def generate_parameter_file(params):
...
+        if variant_name == "boot_l2":
             case_name = params.get("case_name")
             if case_name:
                 kar_cmd += " --%s=%s " % (test_type, case_name)
+        prefix_case_name = "x86_cpu_model"
+        if variant_name == "check_cpu_model_l2":
+            case_name = params.get("case_name", "")
+            for i in host_cpu_models:
+                case_name += "%s.%s," % (prefix_case_name, i)
+            kar_cmd += " --%s=%s " % (test_type, case_name[:-1])
```
Comment (yanan-fu) [inline qemu/tests/nested_test.py:84]: Move `prefix_case_name` to the case `check_cpu_model_l2`.
```
        variant_name = params.get("nested_test")
        case_name = params.get("case_name", '')
        if variant_name == "check_cpu_model_l2":
            host_cpu_models = virttest_cpu.get_host_cpu_models()
            logging.info("host_cpu_model: %s" % host_cpu_models)
            for i in host_cpu_models:
                case_name += "%s.%s," % (prefix_case_name, i)
            case_name.strip(',')
        kar_cmd += " --%s=%s " % (test_type, case_name)
```

Comment (qcheng-redhat) [reply qemu/tests/nested_test.py:83]: Fixed. Thanks

Comment (yanan-fu) [reply qemu/tests/nested_test.py:83]: I do not think it be fixed.
1. `prefix_case_name` in cfg for case `check_cpu_model_l2`
2. No need to add the if block for 'boot_l2'
3. `kar_cmd` is out of all, the format is same for all cases.

Comment (qcheng-redhat) [reply qemu/tests/nested_test.py]: Fixed. Please review. Thanks

File: qemu/tests/cfg/nested_test.cfg (hunk 8 above)
Comment (yanan-fu) [reply qemu/tests/cfg/nested_test.cfg]: @PaulYuuu  i am ok with the current change for this conversation now, How about you ?  We can close this conversation if it is ok for you.

Comment (PaulYuuu) [reply qemu/tests/cfg/nested_test.cfg]: My suggestion is not to add an extra variable `prefix_case_name`. `case_name` should be a general variable for all nested test.
But indeed, we need to filter those not supported test cases for `check_cpu_model_l2`, so reformatting the `case_name` is necessary:
```python
case_name = ','.join(["%s.%s" % (case_name, _) for _ in host_cpu_models])

Comment (yanan-fu) [reply qemu/tests/cfg/nested_test.cfg]: In this way, the `case_name` defined in the cfg is not the real case name. Reformat in the script.
But i am ok with that too, both are ok for me ~

Comment (qcheng-redhat) [reply qemu/tests/cfg/nested_test.cfg:19]: Changed prefix_case_name to case_name.

Comment (yanan-fu) [reply qemu/tests/cfg/nested_test.cfg:19]: ack

File: qemu/tests/nested_test.py [hunk 12]
```diff
@@ -62,13 +63,21 @@ def generate_parameter_file(params):
         guest_password = params.get("password")
 
         bootstrap_options = params.get("nested_bs_options")
+        accept_cancel = params.get("accept_cancel", False)
```
[the remark this answers is not in the document]
Comment (qcheng-redhat) [reply qemu/tests/nested_test.py]: Fixed. Thanks!

**Comment by PaulYuuu** [comment]
ack

**Comment by yanan-fu** [comment]
@qcheng-redhat  Please provide the latest test result, thanks!

**Comment by qcheng-redhat** [comment]
Host CPU: Skylake-Server

When accept_cancel = no

# python3 ./ConfigTest.py --testcase=nested_test --guestname=RHEL.8.2.0 --customsparams="l2_guest_name=RHEL.8.2.0\nl2_kar_options=--machine=q35" --machine=q35 --clone=no

(1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.boot_l2.q35: PASS (162.46 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35: FAIL: ansible_cmd failed, status: 2, output: \nPLAY [deploy kar on L1 guest] **************************************************\n\nTASK [Configure swith] *********************************************************\nok: [10.73.3.191]\n\nTASK [Install git] **************... (680.14 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

When accept_cancel = yes

# python3 ./ConfigTest.py --testcase=nested_test --guestname=RHEL.8.2.0 --customsparams="l2_guest_name=RHEL.8.2.0\nl2_kar_options=--machine=q35" --machine=q35 --clone=no

 (1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.boot_l2.q35: PASS (142.77 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35: PASS (676.63 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Comment by qcheng-redhat** [comment]
screen python3 ./ConfigTest.py --testcase=nested_test --guestname=RHEL.8.2.0 --customsparams="l2_guest_name=RHEL.8.2.0\nl2_kar_options=--machine=q35 --clone=no" --machine=q35 --clone=no --nrepeat=50

python3 /root/kar/workspace/avocado-vt/scripts/scan_results.py
CaseName                                 Status          StartTime       EndTime         Result     TimeElapsed
'01-repeat1.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 11:19:20 2020-04-28 11:33:34 PASS 853.755742
'02-repeat2.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 11:33:34 2020-04-28 11:47:37 PASS 843.024048
'03-repeat3.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 11:47:37 2020-04-28 12:01:38 PASS 841.310411
'04-repeat4.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 12:01:39 2020-04-28 12:15:45 PASS 846.860316
'05-repeat5.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 12:15:45 2020-04-28 12:29:45 PASS 839.7815
'06-repeat6.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 12:29:45 2020-04-28 12:43:43 PASS 838.219082
'07-repeat7.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 12:43:43 2020-04-28 12:57:41 PASS 837.849626
'08-repeat8.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 12:57:41 2020-04-28 13:11:37 PASS 835.766933
'09-repeat9.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 13:11:37 2020-04-28 13:25:35 PASS 838.358567
'10-repeat10.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 13:25:35 2020-04-28 13:39:23 PASS 827.117651
'11-repeat11.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 13:39:23 2020-04-28 13:53:05 PASS 822.831994
'12-repeat12.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 13:53:05 2020-04-28 14:07:02 PASS 836.593345
'13-repeat13.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 14:07:02 2020-04-28 14:20:49 PASS 826.429602
'14-repeat14.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 14:20:49 2020-04-28 14:34:41 PASS 832.289836
'15-repeat15.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 14:34:41 2020-04-28 14:48:37 PASS 836.091384
'16-repeat16.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 14:48:37 2020-04-28 15:02:35 PASS 837.810936
'17-repeat17.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 15:02:35 2020-04-28 15:16:35 PASS 839.873253
'18-repeat18.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 15:16:35 2020-04-28 15:30:30 PASS 834.981859
'19-repeat19.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 15:30:30 2020-04-28 15:44:27 PASS 837.778382
'20-repeat20.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 15:44:28 2020-04-28 15:58:26 PASS 838.691825
'21-repeat21.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 15:58:26 2020-04-28 16:12:18 PASS 831.453673
'22-repeat22.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 16:12:18 2020-04-28 16:26:10 PASS 831.855224
'23-repeat23.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 16:26:10 2020-04-28 16:40:05 PASS 835.905098
'24-repeat24.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 16:40:05 2020-04-28 16:54:02 PASS 836.549727
'25-repeat25.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 16:54:02 2020-04-28 17:07:58 PASS 836.388048
'26-repeat26.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 17:07:58 2020-04-28 17:22:00 PASS 841.563598
'27-repeat27.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 17:22:00 2020-04-28 17:35:52 PASS 831.948485
'28-repeat28.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 17:35:52 2020-04-28 17:49:45 PASS 833.1475
'29-repeat29.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 17:49:45 2020-04-28 18:03:42 PASS 836.349835
'30-repeat30.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 18:03:42 2020-04-28 18:17:41 PASS 838.936194
'31-repeat31.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 18:17:41 2020-04-28 18:31:33 PASS 831.996604
'32-repeat32.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 18:31:33 2020-04-28 18:45:30 PASS 837.248714
'33-repeat33.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 18:45:30 2020-04-28 18:59:29 PASS 838.911851
'34-repeat34.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 18:59:29 2020-04-28 19:13:26 PASS 836.895195
'35-repeat35.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 19:13:26 2020-04-28 19:27:09 PASS 823.18578
'36-repeat36.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 19:27:09 2020-04-28 19:41:10 PASS 840.620628
'37-repeat37.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 19:41:10 2020-04-28 19:54:57 PASS 827.086438
'38-repeat38.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 19:54:57 2020-04-28 20:08:53 PASS 836.305208
'39-repeat39.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 20:08:53 2020-04-28 20:22:40 PASS 826.600522
'40-repeat40.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 20:22:40 2020-04-28 20:36:39 PASS 839.110438
'41-repeat41.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 20:36:39 2020-04-28 20:50:36 PASS 836.866467
'42-repeat42.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 20:50:36 2020-04-28 21:04:32 PASS 836.792427
'43-repeat43.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 21:04:32 2020-04-28 21:18:29 PASS 837.0182
'44-repeat44.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 21:18:29 2020-04-28 21:32:26 PASS 836.593875
'45-repeat45.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 21:32:26 2020-04-28 21:46:24 PASS 837.995429
'46-repeat46.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 21:46:24 2020-04-28 22:01:14 PASS 890.120343
'47-repeat47.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 22:01:14 2020-04-28 22:15:08 PASS 834.052685
'48-repeat48.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 22:15:08 2020-04-28 22:29:08 PASS 839.937111
'49-repeat49.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 22:29:08 2020-04-28 22:43:07 PASS 839.111971
'50-repeat50.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.nested_test.check_cpu_model_l2.q35' Finshed 2020-04-28 22:43:07 2020-04-28 22:57:02 PASS 834.504612
Summary: 
Finshed=50, PASS=50

**Comment by yanan-fu** [comment]
Thanks all!

**Pull request merged**